swift_browser_ui_frontend/src/entries/main.js

```diff
@@ -44,6 +44,21 @@ checkIDB().then(result => {
   }
 });
 
+if ("serviceWorker" in navigator) {
```

Comment:
I'd guess a flag would be better – you wouldn't have access to uploading due to unsupported browser but no reason to prevent other usage for those without it. Adding that would be a good thing though.

Reply:
I'll force a disable in the upload view with a message in cases it doesn't have access to the serviceworker.